docs/blocks-storage/querier.md

```diff
@@ -136,10 +136,6 @@ querier:
   # CLI flag: -querier.query-store-for-labels-enabled
   [query_store_for_labels_enabled: <boolean> | default = false]
 
-  # Enable the @ modifier in PromQL.
-  # CLI flag: -querier.at-modifier-enabled
-  [at_modifier_enabled: <boolean> | default = false]
```

Comment:
Are we okay with totally removing this? This is a breaking change which is not backward compatible.

Reply:
This change will indeed lead to incompatibility, if we want to be compatible, we need to keep both CLI flag and configuration option, the documentation can be cleaned up completely.

But this function should be used by fewer people, We determine whether to be compatible or make an incompatibility notice？

Reply:
It is more a question to our maintainers 😄 @alanprot @alvinlin123

Reply:
What would be the reason to disable this feature? 

I think I'm ok with it but we could also mark as deprecated and remove on the 1.15.

@alvinlin123 WDYT?

Reply:
@songjiayang do you know what happens if we remove this option, but user configuration still have this option? Would Cortex still load? If it does, then I am fine removing it, else maybe we should make it an no-op option just like prometheus?

Reply:
when remove `-querier.at-modifier-enabled` flag and `at_modifier_enabled ` configuration, run cortex will failed:

```
 $ docker logs play-with-cortex-cortex-1-1
flag provided but not defined: -querier.at-modifier-enabled
Run with -help to get list of available parameters

docker logs play-with-cortex-cortex-1-1
error loading config from /cortex/config/cortex.yaml: Error parsing config file: yaml: unmarshal errors:
  line 40: field at_modifier_enabled not found in type querier.Config
```

I will remain the flag and conriguration, only make them to true.

Reply:
Can you mark it as deprecated? Just do something like

https://github.com/cortexproject/cortex/blob/6249935727fab9f80496370c68d1a3eba8d26de9/pkg/util/validation/limits.go#L154

Reply:
yes, I copy from prometheus `"[DEPRECATED] Enable the @ modifier in PromQL.`